### PR TITLE
missing tests

### PR DESCRIPTION
**Pull request by tezc**

### CODE REVIEW

File: tests/unit/pubsub.tcl [hunk 1]
```diff
@@ -423,4 +423,9 @@ start_server {tags {"pubsub network"}} {
         assert_equal "pmessage * __keyevent@${db}__:new bar" [$rd1 read]
         $rd1 close
     }
+
+    test {PUBSUB HELP should not have unexpected options} {
```
Comment (guybe7) [inline tests/unit/pubsub.tcl]: we actually want to non-error response...

can you please gather all HELP tests in one test function?

also, please add
```
client|help 
cluster|help 
command|help 
config|help 
function|help
```

something like
```
assert_match "*CLIENT <subcommand> *" [r CLIENT HELP]
```

Comment (tezc) [reply tests/unit/pubsub.tcl]: done.
`cluster help` wouldn't work without cluster-enabled config. I had to put it into another file.

File: tests/unit/type/string.tcl [hunk 2]
```diff
@@ -459,6 +459,17 @@ start_server {tags {"string"}} {
             assert_equal [string range $bin $_start $_end] [r getrange bin $start $end]
         }
     }
+
+    test "SUBSTR fuzzing" {
```
Comment (guybe7) [inline tests/unit/type/string.tcl]: i think we just want a basic test for coverage.. no need for fuzzing

Comment (tezc) [reply tests/unit/type/string.tcl]: Looks like SUBSTR is deprecated in favor of GETRANGE. So, I copied this test from the just above GETRANGE test. 
Anyway, changed it with some basic test.

**Pull request merged**